.pipelines/release-agent.sh

```diff
@@ -4,11 +4,11 @@
 
 # These are plain pipeline variable which can be modified anyone in the team
```

Comment:
I don't think we are using this file anymore. Please confirm and if true remove this file

Reply:
this is the script for manually push to mcr path for arc release, we might need it for reference